### ISSUE TITLE
CI/CD and dev environment setup scripts

### ISSUE THREAD

**Issue by jazzybluesea** [openenclave/openenclave#2138]
Currently in windows the install script to setup the drivers/dependencies is separate from the CI/CD logic.  In order words, we have the same logic in two different places.  It would be nice to one golden reference where this logic resides.

**Comment by jhand2** [comment]
This setup script only works for installing SGX-specific dependencies

**Comment by BRMcLaren** [comment]
@jhand2 Yes, this is correct. We currently do not support a non SGX enabled windows VM, is my understanding.
